### DISCUSSION TITLE
Google Calendar 404

### DISCUSSION THREAD

**Discussion by dietrichmd** [liedekef/events-made-easy#187] [Q&A]
For years, the Google Calendar link/snippet found at https://www.e-dynamics.be/wordpress/faq/ worked fine.  Specifically
```
<a target="_blank" href="http://www.google.com/calendar/event?action=TEMPLATE
&text=#URL_NAME
&dates=#_{Ymd}T#H#_{i}00/#@_{Ymd}T#@H#@_{i}00
&sprop=#URL_EVENTPAGEURL
&location=#URL_ADDRESS,+#URL_TOWN
&details=#URL_NOTES"
>Add to Google Calendar</a>
```
However, recently that code is returning a 404 from Google.  Any ideas?

Thanks Franky.

**Comment by liedekef** [comment]
If you have it like you wrote it (meaning several things on separate lines), then it is logical. Make sure it is all on one line and try again (and use https, not http anymore)

**Comment by dietrichmd** [comment]
it's all one line.  ill swap to https and see what happens.

**Comment by liedekef** [comment]
I tried it here, works just fine ...

**Comment by dietrichmd** [comment]
Perfect.  https:// was, indeed, the problem.  Thanks :)
